bin/metrics.js

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var fs = require('fs')
+var section
+
+module.exports = {
+  run: run,
+  countAccounts: [
+    'SELECT COUNT(*) AS count',
+    'FROM accounts',
+    'WHERE createdAt < ?;'
+  ].join('\n'),
+  countVerifiedAccounts: [
+    'SELECT COUNT(*) AS count',
+    'FROM accounts',
+    'WHERE createdAt < ?',
+    'AND emailVerified = true;'
+  ].join('\n'),
+  countAccountsWithTwoOrMoreDevices: [
+    'SELECT COUNT(*) AS count',
+    'FROM (',
+    '    SELECT a.uid',
+    '    FROM accounts AS a',
+    '    INNER JOIN sessionTokens AS s',
+    '    ON a.uid = s.uid',
+    '    WHERE a.createdAt < ?',
+    '    GROUP BY (a.uid)',
+    '    HAVING COUNT(s.tokenId) > 1',
+    ') AS sub;'
+  ].join('\n'),
+  countAccountsWithThreeOrMoreDevices: [
+    'SELECT COUNT(*) AS count',
+    'FROM (',
+    '    SELECT a.uid',
+    '    FROM accounts AS a',
+    '    INNER JOIN sessionTokens AS s',
+    '    ON a.uid = s.uid',
+    '    WHERE a.createdAt < ?',
+    '    GROUP BY (a.uid)',
+    '    HAVING COUNT(s.tokenId) > 2',
+    ') AS sub;'
+  ].join('\n'),
+  countAccountsWithMobileDevice: [
+    'SELECT COUNT(DISTINCT a.uid) AS count',
+    'FROM accounts AS a',
+    'INNER JOIN sessionTokens AS s',
+    'ON a.uid = s.uid',
+    'WHERE a.createdAt < ?',
+    'AND s.uaDeviceType = \'mobile\';'
+  ].join('\n')
+}
+
+if (require.main === module) {
+  module.exports.run(
+    fs.readFileSync(
+      process.argv[2] || '/etc/gather_basic_metrics.conf',
+      { encoding: 'utf8' }
+    )
+    .split('\n')
+    .map(trim)
+    .filter(filterConfig)
+    .reduce(reduceConfig, {})
+  )
```

Comment:
Factoring nit: I suggest encapsulating this `split->map->filter->reduce` chain into an exportable function like `parseConfigFile` and putting all of them underneath the new comment you added explaining what it's for.